Source/WebCore/svg/SVGLocatable.cpp

```diff
@@ -25,29 +25,37 @@
 
 #include "RenderElement.h"
 #include "SVGElementTypeHelpers.h"
+#include "SVGForeignObjectElement.h"
 #include "SVGGraphicsElement.h"
 #include "SVGImageElement.h"
 #include "SVGLayerTransformComputation.h"
 #include "SVGMatrix.h"
 #include "SVGNames.h"
+#include "SVGSVGElement.h"
+#include "SVGSymbolElement.h"
 #include "TransformState.h"
 
 namespace WebCore {
 
-static bool isViewportElement(Node* node)
+static SVGElement* toViewportElement(Element& element)
```

Comment:
I think this is a little bit confusing. Someone would expect `toViewportElement` has to convert the `Element` to `ViewportElement` which does not exist. Also SVGElement has a method called `SVGElement::viewportElement()`.

Reply:
I don't think I made it much worse given that the original name was `isViewportElement()`. I'd really like to address in this PR so I'm open to suggestions on a better name.

Reply:
I updated my PR, I found a way to keep isViewportElement() mostly as-is while still optimizing what I wanted to.